### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by punnarpulusu**
**1. Issue, if available:**

Updating typo in the documentation

**2. Description of changes:**

Updating typo in the documentation

**3. How was this change tested?**

Small typo -- no operational change

**4. Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [x ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by chrisnegus — COMMENTED** [review]
There is an expression to "shave seconds off". But since it is sort of a slang expression, I think we should accept your change.

File: website/content/en/v0.6.4/concepts/_index.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Instead, it relies on the kube-scheduler to make the same scheduling decision af
 A node that Karpenter launches has its pods bound immediately.
 The kubelet doesn't have to wait for the scheduler or for the node to become ready.
 It can start preparing the container runtime immediately, including pre-pulling the image.
-This can shave seconds off of node startup latency.
+This can save seconds off of node startup latency.
```
Comment (ellistarn) [inline website/content/en/v0.6.4/concepts/_index.md:125]: Can you add this to the `preview` docs, as this is what we use to generate the next version.

Comment (punnarpulusu) [reply website/content/en/v0.6.4/concepts/_index.md:125]: Updated preview docs.

**Review by ellistarn — APPROVED** [review]
thank you!

**Pull request merged**